Handle open issue 448 by raising an exception

Dylan1496 opened the pull request:
Solves https://github.com/actionless/pikaur/issues/448 by raising an exception when pikaur -P calls makepkg --printsrcinfo -p PKGBUILD via systemd-run. Perhaps not the most elegant solution.

actionless commented on pikaur/core.py:
```diff
@@ -204,6 +204,8 @@ def isolate_root_cmd(cmd: List[str], cwd=None) -> List[str]:
     ):
         if os.environ.get(env_var_name) is not None:
             base_root_isolator += ['-E', f'{env_var_name}={os.environ[env_var_name]}']
+    if cmd == ['makepkg','--printsrcinfo','-p','PKGBUILD']:
```
also please add a `@TODO:` comment explaining what it's only a workaround

actionless commented:
and make the CI pass :)

Dylan1496 commented:
Added comments regarding the fix, and fixed the errors from flake. Now it is failing at the pylint phase, because it can't find tomli. However on my machine pylint found no errors in the updated core.py file.

actionless commented:
@Dylan1496 please rebase on master, i pushed the workaround to the broken pylint arch package

actionless commented:
thanks!
